RadixWallet/Features/HomeFeature/Coordinator/Home+View.swift

```diff
@@ -99,23 +99,19 @@ extension Home {
 			}
 		}
 
-		private struct HeaderView: SwiftUI.View {
+		struct HeaderView: SwiftUI.View {
+			let store: StoreOf<Home>
+
 			var body: some SwiftUI.View {
-				VStack(alignment: .leading, spacing: .small2) {
+				VStack(spacing: .small2) {
 					Text(L10n.HomePage.title)
 						.foregroundColor(.app.gray1)
 						.textStyle(.sheetTitle)
+						.flushedLeft
+						.padding(.horizontal, .medium1)
 
-					HStack {
-						Text(L10n.HomePage.subtitle)
```

Comment:
Confirm if we want to remove subtitle.

Reply:
It's explicitly mentioned in the ticket. 

The ticket also says "After the carousel is in place, the Start using Radix Wallet CTA card from home screen can be moved up in the carousel. It is not yet clear if it will have the same design, but can be used as an example." so that should not be overlooked, when the design is finalised.